src/ecrc-frontend/src/components/page/applicationForm/ApplicationForm.js

```diff
@@ -372,24 +418,47 @@ export default function ApplicationForm({
     type: "submit"
   };
 
+  const validatePhoneNumber = phone => {
+    const re = /1?[ \-(]*\d{3}[ \-)]*\d{3}[ -]*\d{4}/;
+    return re.test(phone);
+  };
+
+  const validateEmail = emailTxt => {
+    const re = /[a-z0-9!#$%&'*+/=?^_‘{|}~-]+(?:\.[a-z0-9!#$%&'*+/=?^_‘{|}~-]+)*@(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?\.)+[a-z0-9](?:[a-z0-9-]*[a-z0-9])?/;
+    return re.test(emailTxt);
+  };
+
+  const validatePostalCode = postalCode => {
+    const re = /[A-Z][0-9][A-Z][ -]*[0-9][A-Z][0-9]/;
+    return re.test(postalCode.toUpperCase());
+  };
+
```

Comment:
wondering if using a lib like `validate.js` was considered for this use case... it may be easier than defining custom regex for every validation type required

Reply:
I looked into it a little but wasn't sure if we wanted/needed an entire lib for 3 validations. If you think is worth should be able to set it up.

Reply:
if it is only these three we need and no scope for multiple reuse then definitely not worth the lib :) just wanted to ask - if it was already considered and decided to not be used then that's great